clippy_lints/src/attrs/useless_attribute.rs

```diff
@@ -25,6 +25,10 @@ pub(super) fn check(cx: &LateContext<'_>, item: &Item<'_>, attrs: &[Attribute])
                                 || is_word(lint, sym!(unreachable_pub))
                                 || is_word(lint, sym!(unused))
                                 || is_word(lint, sym!(unused_import_braces))
+                                || is_word(lint, sym!(unused_braces))
+                                || is_word(lint, sym!(dead_code))
```

Comment:
The CI is currently failing due to this:
```suggestion
                                || is_word(lint, rustc_span::sym::dead_code)
```